content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/README.md

```diff
@@ -115,3 +131,6 @@ Example:
 * **Co-authors**: [Jean-Christophe Kautzmann](https://github.com/jckautzmann), [Richard Hand](https://github.com/richardhand), [Vlad Bailescu](https://github.com/vladbailescu)
 
 _If you were involved in the authoring of this component and are not credited above, please reach out to us on [GitHub](https://github.com/adobe/aem-core-wcm-components)._
+
+
+[yt-parameters]: https://developers.google.com/youtube/player_parameters
```

Comment:
I would prefer these are documented in `apps/core/wcm/components/embed/v1/embed/embeddable/youtube/README.md` as they are specific to this embeddable.